src/v0/destinations/active_campaign/util.js

```diff
@@ -1,31 +1,24 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
 const {
-  nodeSysErrorToStatus,
   getDynamicErrorType,
+  processAxiosResponse,
 } = require('../../../adapters/utils/networkUtils');
 const tags = require('../../util/tags');
 
-const errorHandler = (err, message) => {
-  if (err.response) {
-    throw new NetworkError(
-      `${message} (${err.response?.statusText},${JSON.stringify(err.response?.data)})`,
-      err.status,
-      {
-        [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(err.status),
-      },
-      err,
-    );
-  } else {
-    const httpError = nodeSysErrorToStatus(err.code);
-    throw new NetworkError(
-      `${message} ${httpError.message}`,
-      httpError.status,
-      {
-        [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(httpError.status),
-      },
-      err,
-    );
+const errorHandler = (httpCallError, message) => {
+  const {response, status} = processAxiosResponse(httpCallError);
```

Comment:
Is this case `const httpError = nodeSysErrorToStatus(err.code);` handled in processAxiosResponse ?

Reply:
Yes. Please check [here](https://github.com/rudderlabs/rudder-transformer/blob/develop/src/adapters/utils/networkUtils.js#L134)